src/ranking_watcher/usma.rs

```diff
@@ -11,13 +11,15 @@ const RANKING_URL: &str = "https://ranking.cvgo.re/";
 
 type Result<T> = result::Result<T, Box<dyn Error + Send + Sync>>;
 
-#[derive(Debug, Eq, PartialEq)]
+#[derive(Clone, Debug, Eq, PartialEq)]
```

Comment:
Why not `Copy`?

Reply:
Got rid of it altogether by extracting changed entries to vec of refs instead.